bash-completion/stenc

```diff
@@ -26,7 +30,7 @@ _stenc () {
             COMPREPLY=($(compgen -W 'off on mixed' -- "$cur"))
             return
             ;;
-        -k | --keyfile )
```

Comment:
What do you mean? This is a correction for the bash completion script because the actual option name is `--key-file` but it is incorrectly called `keyfile` here.

Reply:
Oh I see. never mind.